src/clr-angular/modal/modal.ts

```diff
@@ -41,36 +39,6 @@ import { GHOST_PAGE_ANIMATION } from './utils/ghost-page-animations';
       transition('void => *', [style({ opacity: 0 }), animate('0.2s ease-in-out', style({ opacity: 0.85 }))]),
       transition('* => void', [animate('0.2s ease-in-out', style({ opacity: 0 }))]),
     ]),
```

Comment:
do we care about trailing commas in arrays?

Reply:
At times I take them off, but I've seen them in our code base elsewhere too so I assume we don't care?

Reply:
I don't think the browsers we support have a problem with them. Still feels messy. I've already approved, so my feelings on it are clear. It pings my OCD. But not worth blocking progress.

Reply:
This is something we could enforce / standardize the codebase to with prettier: https://prettier.io/docs/en/options.html#trailing-commas